homeassistant/components/ssdp/__init__.py

```diff
@@ -164,6 +164,22 @@ def get(self, name: str, default: Any = None) -> Any:
             return getattr(self, name)
         return self.upnp.get(name, default)
 
+    def __iter__(self) -> Iterator[str]:
```

Comment:
This is required for `in` calls like in konnected:
https://github.com/home-assistant/core/blob/fb909eca8a643447f0a8355c0125a053a58ce77f/homeassistant/components/konnected/config_flow.py#L253-L256

As in https://github.com/home-assistant/core/pull/60334/files#r756924593, this might need to be added to `zeroconf`/`dhcp`/`usb`/`mqtt`

Reply:
We probably should since its a temp measure and the goal is to avoid a breaking change